modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -930,6 +1337,10 @@ private JSONArray _createObjectEntriesJSONArray(
 		return jsonArray;
 	}
 
+	private String _escape(String string) {
```

Comment:
What do you think about creating a unique method `_getFilter`, something like:

```
private String _getFilter(String filterExpr, Object... elements) {
	return URLCodec.encodeURL(String.format(filterExpr, elements));
}
```

and the usage:

```
_assertFilter(
	_OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_1,
	_getFilter("%s/%s eq '%s'", _objectRelationship1.getName(),
			_OBJECT_FIELD_NAME_2, _OBJECT_FIELD_VALUE_2),
	_objectDefinition1);
```